### DISCUSSION TITLE
I can scan the crazyflie, but not connect to it

### DISCUSSION THREAD

**Discussion by socamo7005** [bitcraze/discussions#1334] [General]
I am trying to connect to the crazyflie via the virtual machine, I am able to scan the radio with the crazyradio PA, and able to click on connect. The M4 light starts flashing green, but the virtual machine says it is still connecting to the radio, it stays like that, and I get no data from the drone. I would appreciate the help.

**Comment by Residualstress** [comment]
Hi, have you made any changes to the Crazyflie? Usually, I met such problem when I flashed a wrong version of the firmware, i.e., flash the cf2 firmware to the Bolt, or revised some files in the firmware and flash it to the Crazyflie.

**Comment by ToveRumar** [comment]
Hi!

Could you give us some more specifications on your setup?  
What OS and version are you using? 

To get some more information on your issue could you answer the following?
- Are you flashing the latest FW through the cfclient? 
- Can you see any issues when looking at the log output from the cfclient?
- Is it possible to connect and communicate with the Crazyflie over usb?
